tests/modules/nf-core/sourmash/compare/main.nf

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { SOURMASH_SKETCH } from '../../../../../modules/nf-core/sourmash/sketch/main.nf'
+include { SOURMASH_COMPARE } from '../../../../../modules/nf-core/sourmash/compare/main.nf'
+
+workflow test_sourmash_compare {
+    
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+    ]
+
+    save_numpy_matrix = true
+    save_csv = true
+    
+    SOURMASH_SKETCH ( input )
+
+    SOURMASH_COMPARE (
+        SOURMASH_SKETCH.out.signatures.collect { it[1] },
```

Comment:
```suggestion
        SOURMASH_SKETCH.out.signatures,
```

Reply:
How do I specify that sourmash compare should run on all of the signatures that come out of `SOURMASH_SKETCH` in a single time? This is the stdout from sourmash compare to give you more of an intuition for what it's doing:
```
loaded 6 signatures total.

0-SRR19888423       	[1.    0.31  0.288 0.298 0.25  0.201]
1-SRR19888427       	[0.31  1.    0.199 0.235 0.207 0.185]
2-SRR19888432       	[0.288 0.199 1.    0.451 0.308 0.23 ]
3-SRR19888434       	[0.298 0.235 0.451 1.    0.46  0.304]
4-SRR19888438       	[0.25  0.207 0.308 0.46  1.    0.38 ]
5-SRR19888440       	[0.201 0.185 0.23  0.304 0.38  1.   ]
```

Reply:
This is coming back to the paradox about `meta` and `signatures`. Since you are actually comparing multiple samples, we can regard sourmash compare as a summarizing action. So in that case you need to collect the signatures as you had before but drop the `meta` from the input. So:

```nextflow
SOURMASH_COMPARE(SOURMASH_SKETCH.out.signatures.collect { it[1] })
```

and


```nextflow
input:
path signatures
```

Reply:
Yay it's working!!!!!!! Thank you so much for your help! This was tricky for me but I understand it much better now. I like have the wording for "summarizing action" -- that was super helpful. 

Still some cleanup to do but super excited pytests are passing!